api/types/auth.go

```diff
@@ -6,26 +6,24 @@ package types
 
 // AuthRequest contains authorization information for connecting to a envd server.
 type AuthRequest struct {
```

Comment:
Maybe can use authn,separate from authz to avoid confusion.

Reply:
LGTM